ktxspec.adoc

```diff
@@ -871,35 +871,43 @@ A _dfDescriptor_ is useful in the following cases:
 The following restrictions must be obeyed when setting the fields of a
 _dfDescriptorBlock_.
 
-* The value of `<<_vkformat,vkFormat>>` and the image data must
-  never contradict the DFD and, vice-versa, the DFD must not
-  contradict `<<_vkformat,vkFormat>>`. Thus the value of `vkFormat`
-  can be used as a shortcut. Only when it is `VK_FORMAT_UNDEFINED`,
-  does an application need to utilitize sample information from the
-  DFD.  (In this case contradiction is impossible.)
-* `colorModel` must be `KHR_DF_MODEL_RGBSDA`
-  or one of the block compressed color models listed in <<KDF13>>
-  Section 5.6 or its successors, currently `KHR_DF_MODEL_BC1A` to
-  `KHR_DF_MODEL_UASTC`.
-* `colorPrimaries` may be any of the available values since
-  conversion of the selected primaries and white point to a display's
-  can be done simply with a 3x3 matrix multiply.
+* If `<<_vkformat,vkFormat>>` is not `VK_FORMAT_UNDEFINED`, DFD's
+  `texelBlockDimension*`, `bytesPlane*`, and sample information fields
+  must match the format's definition. The `colorModel` must be
+  `KHR_DF_MODEL_RGBSDA` or the matching block compressed color model
+  listed in <<KDF13>> Section 5.6 or its successors, currently
+  `KHR_DF_MODEL_BC1A` to `KHR_DF_MODEL_UASTC`.
+* If `<<_vkformat,vkFormat>>` is one of the `+*_SRGB{,_*}+` formats,
+  `transferFunction` must be `KHR_DF_TRANSFER_SRGB`.
+* If `<<_vkformat,vkFormat>>` is not one of the `+*_SRGB{,_*}+` formats
+  and an sRGB variant of that format exists, `transferFunction` should not
```

Comment:
I think s/should not/must not/. Why di d you choose "should"?

Reply:
If future Vulkan versions would have an equivalent of `MTLPixelFormatBGRA10_XR_sRGB`, KTX files created before that with `_UNORM` and sRGB DFD should not suddenly become invalid.